README.md

```diff
@@ -0,0 +1,93 @@
+# Rails React Hello World App
+
+In this project I developed a simple app to connect react with rails and fetch some random information from database.
+
+## Screenshots
+
+![image](./app/assets/images/Screenshots/Capture.png)
+
```

Comment:
- [ ] Currently, the image is broken. Please fix it to make the README professional.

Reply:
Addressed this